packages/teleterm/src/services/pty/ptyHost/buildPtyOptions.ts

```diff
@@ -5,6 +5,7 @@ import {
   ResolveShellEnvTimeoutError,
 } from './resolveShellEnv';
 import { PtyCommand, PtyProcessCreationStatus } from '../types';
+import { delimiter } from 'path';
 
 export async function buildPtyOptions(
```

Comment:
Connect sets the default working directory for new shell sessions on Windows to the directory where the app is installed. It seems that we should default it to the user home directory instead, just as cmd and powershell do.

Reply:
I'll address it in the next PR as discussed.